.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
Not sure what libraries I should put in pyproject.toml and test.py for 3.11 and 3.12.